addon/components/horizontal-bar-chart.js

```diff
@@ -40,6 +42,60 @@ export default ChartComponent.extend(FloatingTooltipMixin,
   // Layout
   // ----------------------------------------------------------------------------
 
+  /**
+  * Overrides values in addon/mixins/axis-titles.js
+  * @override
+  */
+  xAxisPositionX: Ember.computed('graphicWidth', 'labelWidthOffset', function(){
+    return this.get('graphicWidth') / 2;
+  }),
+
+  /**
+  * X Axis Titles needs extra padding, else will intersect with the lowest bar
+  * @override
+  */
+  xAxisPositionY: Ember.computed('graphicBottom', 'axisPadding', function(){
+    return this.get('graphicBottom') + this.get('xAxisPadding');
+  }),
+
+  /**
+  * @override
+  */
+  yAxisPositionX: Ember.computed('graphicBottom', 'axisPadding', function(){
```

Comment:
Why do we depend on `graphicBottom` and `axisPadding` when  `graphicHeight`  is the only one we are using?